.github/workflows/render-rmds.yml

```diff
@@ -0,0 +1,37 @@
+name: Test render notebooks
+# Before merging notebooks, check that they render
+
+# This workflow only runs when it is manually dispatched
+on:
+  pull_request:
+    branches: [ master ]
+    paths: 
+      - '**.Rmd'
+      - 'make-live.R'
```

Comment:
Would this work without the full path?
```suggestion
      - 'scripts/make-live.R'
```

The check is running, but there's a change to the GSVA `.Rmd` file.

Reply:
I thought it might work either way (matches file names regardless of path), but reading the docs more carefully, it does seem to require the full path. I don't know why I thought differently.